etl/steps/data/garden/emdat/2022-11-24/shared.py

```diff
@@ -0,0 +1,474 @@
+        If None (or an empty dictionary), the region should be in OWID's countries-regions dataset.
+        If not None, it should be a dictionary with any (or all) of the following keys:
+        - "regions_included": List of regions whose countries will be included.
+        - "regions_excluded": List of regions whose countries will be excluded.
+        - "countries_included": List of additional individual countries to be included.
+        - "countries_excluded": List of additional individual countries to be excluded.
+        NOTE: All regions and countries defined in this dictionary should be in OWID's countries-regions dataset.
+
+    Returns
+    -------
+    countries : list
+        List of countries in the specified region.
+
+    """
+    if region_modifications is None:
+        region_modifications = {}
+
+    # Check that the fields in the regions_modifications dictionary are well defined.
+    expected_fields = ["regions_included", "regions_excluded", "countries_included", "countries_excluded"]
+    assert all([field in expected_fields for field in region_modifications])
+
+    # Get lists of regions whose countries will be included and excluded.
+    regions_included = region_modifications.get("regions_included", [region])
+    regions_excluded = region_modifications.get("regions_excluded", [])
+    # Get lists of additional individual countries to include and exclude.
+    countries_included = region_modifications.get("countries_included", [])
+    countries_excluded = region_modifications.get("countries_excluded", [])
+
+    # List countries from the list of regions included.
+    countries_set = set(
+        sum([geo.list_countries_in_region(region_included) for region_included in regions_included], [])
+    )
+
+    # Remove all countries from the list of regions excluded.
+    countries_set -= set(
+        sum([geo.list_countries_in_region(region_excluded) for region_excluded in regions_excluded], [])
+    )
+
+    # Add the list of individual countries to be included.
+    countries_set |= set(countries_included)
+
+    # Remove the list of individual countries to be excluded.
+    countries_set -= set(countries_excluded)
+
+    # Convert set of countries into a sorted list.
+    countries = sorted(countries_set)
+
+    return countries
+
+
+def load_population() -> pd.DataFrame:
+    """Load OWID population dataset, and add historical regions to it.
+
+    Returns
+    -------
+    population : pd.DataFrame
+        Population dataset.
+
+    """
+    # Load population dataset.
+    population = catalog.Dataset(DATA_DIR / "garden/owid/latest/key_indicators/")["population"].reset_index()[
+        ["country", "year", "population"]
+    ]
+
+    # Add data for historical regions (if not in population) by adding the population of its current successors.
+    countries_with_population = population["country"].unique()
+
+    # Consider regions and historical regions.
+    regions = deepcopy(REGIONS)
```

Comment:
Nit - these two lines could be `regions = dict(**REGIONS, **HISTORIC_TO_CURRENT_REGION)`

Reply:
Nice one!